release-notes/swan-lake-2201.1.0-release-note.md

```diff
@@ -266,6 +266,9 @@ To view bug fixes, see the [GitHub milestone for Swan Lake 2201.1.0](https://git
 
 ### Deployment updates
 
+#### New Features
+- Added `name` field for `cloud.config.files` in `Cloud.toml` to change the name of the config map generated
```

Comment:
```suggestion
- Added the `name` field for the `cloud.config.files` property in the `Cloud.toml` file to change the name of the generated config map 
```
Should `cloud.config.files` be `cloud.config` files?

Reply:
cloud.config.files is correct